src/librustdoc/lib.rs

```diff
@@ -587,6 +587,9 @@ fn opts() -> Vec<RustcOptGroup> {
                 "[unversioned-shared-resources,toolchain-shared-resources,invocation-specific]",
             )
         }),
+        unstable("show-type-layout", |o| {
+            o.optflag("", "show-type-layout", "Include the memory layout of types in the docs")
+        }),
```

Comment:
I only added a CLI flag because I think adding an attribute in addition would be overkill.